ocaml/.depend

```diff
@@ -130,13 +130,25 @@ utils/strongly_connected_components.cmx : \
     utils/strongly_connected_components.cmi
 utils/strongly_connected_components.cmi : \
     utils/identifiable.cmi
+utils/target_system.cmo : \
+    utils/misc.cmi \
+    utils/config.cmi \
+    utils/target_system.cmi
+utils/target_system.cmx : \
+    utils/misc.cmx \
+    utils/config.cmx \
+    utils/target_system.cmi
+utils/target_system.cmi :
 utils/targetint.cmo : \
+    utils/numbers.cmi \
```

Comment:
(I don't think we care, but the part of this file's
diff starting here is probably not related to this
pull request.)

Reply:
I reverted part of the diff